### PR TITLE
Posts: Interactive Book Integration Phase 1 Report

### PR DESCRIPTION
**Pull request by manjotsidhu**
## Phase 1 report for Interactive Book Integration

Deploy Preview: https://deploy-preview-76--cvblog.netlify.app/

### CODE REVIEW

**Review by Shivansh2407 — COMMENTED** [review]
LGTM @manjotsidhu :tada: Great work :+1: 
It would be better if you could add an author page (content and assets for the same accordingly) for yourself as this is your first time writing a blog on CV Blog Website, and [this](https://deploy-preview-76--cvblog.netlify.app/author/manjot-sidhu) shows 404 Error currently.

**Comment by manjotsidhu** [comment]
> LGTM @manjotsidhu 🎉 Great work 👍
> It would be better if you could add an author page (content and assets for the same accordingly) for yourself as this is your first time writing a blog on CV Blog Website, and [this](https://deploy-preview-76--cvblog.netlify.app/author/manjot-sidhu) shows 404 Error currently.

Sure.

**Review by Nitish145 — APPROVED** [review]
LGTM ✅

File: content/author/manjot-sidhu.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title = "Manjot Singh Sidhu"
+image = "/images/authors/mss.jpg"
+description = "Full Stack Developer"
++++
+
+{{< rawhtml >}}
+
+<p align="center">
+  <code>
+    <a href="mailto:manjot.techie@gmail.com" title="Gmail"><img width="22" alt="Gmail" src="https://github.com/manjotsidhu/manjotsidhu/raw/master/icons/Gmail.png"> Gmail</a></code>
```
Comment (Nitish145) [inline content/author/manjot-sidhu.md:11]: Alignment is a bit messed up here, please check!

Screenshot 2021-07-09 at 3 10 45 PM

**Review by Nitish145 — CHANGES_REQUESTED** [review]

File: content/author/manjot-sidhu.md (hunk 1 above)
Comment (manjotsidhu) [reply content/author/manjot-sidhu.md:11]: Hmm, consider it as a floating style icon 🤣. It's the same in my GitHub profile.
image

Comment (manjotsidhu) [reply content/author/manjot-sidhu.md:11]: I am lazy to fix that

File: content/author/manjot-sidhu.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
++++
+title = "Manjot Singh Sidhu"
```
Comment (Shivansh2407) [inline content/author/manjot-sidhu.md:2]: You can either update Manjot Sidhu here or in content/posts/Manjot_gsoc_phase_one_2021.md author -> Manjot Singh Sidhu, as the blog is not able to recognise the blogs written by you on the author's page

**Review by Shivansh2407 — CHANGES_REQUESTED** [review]
Please update the PR accordingly and we'll be good to go. Great job @manjotsidhu :tada:

File: content/author/manjot-sidhu.md (hunk 2 above)
Comment (manjotsidhu) [reply content/author/manjot-sidhu.md:2]: Sure

**Comment by manjotsidhu** [comment]
Kindly squash and merge.

**Review by Shivansh2407 — APPROVED** [review]
Working well :tada: @manjotsidhu :+1:

**Comment by tachyons** [comment]
Merging will be only after evaluation is done by the mentors and results are officially announced by the google. Kindly wait

**Pull request merged**